New blog post on clients as product owners. Ready for review (#212)
ombulabs/blog: _posts/2019-11-27-making-your-clients-better-product-owners.markdown

---
layout: post
title: "Making Your Clients Better Product Owners"
date: 2019-11-27 12:30:00
categories: ["Scrum", "Agile"]
author: "abizzinotto"
---

> Different clients bring different projects, perspectives, workflows and experiences, as well as different challenges. Before starting a project, one of those challenges is to define who will be the Product Owner. 

Author: @etagwerker the best definition I know is the one from Scrum.org here https://www.scrumguides.org/scrum-guide.html#team-po however that's a Scrum Guide with a lot of other concepts as well (although that links goes straight to the PO one). Would that be good enough?

Ideally, you would be able to assign the role internally or to the client based solely on the characteristics of the project. However, that's not always the case. It might just be that the client insists on being the Product Owner or that you are a small team and can't really assign the role internally. Whatever the reason, you might end up in a situation where your client isn't really a good Product Owner.

Here I'll share some strategies we implement to help our clients become better Product Owners and ensure the best experience for them and for our team.

<!--more-->

> ##Communicate expectations clearly

Reviewer: @ABizzinotto Can you add a space after the `##` here and in all the titles? Otherwise the titles won't show I think. You can check the preview https://github.com/ombulabs/blog/blob/2aa7a3098639bc1c78119115c804a19595edc9ae/_posts/2019-11-27-making-your-clients-better-product-owners.markdown

Author: Done

> Not every client is familiar with the responsibilities of the Product Owner. Not every client is even familiar with Scrum as a whole. Therefore, it is important to communicate expectations clearly at the beginning of the project.

Reviewer: @ABizzinotto Maybe you can add a link when you mention `Scrum`

Author: @lubc do you mean a link like this? https://www.scrumguides.org/scrum-guide.html#definition

Author: @lubc actually, we ended up replacing 'Scrum' :)

> Not every client is familiar with the responsibilities of the Product Owner. Not every client is even familiar with Scrum as a whole. Therefore, it is important to communicate expectations clearly at the beginning of the project.

Author: Done. I just capitalized Agile and Lean

> During the kick-off call, make sure the person on the client's team assigned to the project understands what the Scrum master and the Scrum team expect from them. Some key points to cover:

Reviewer: @ABizzinotto When you say `kick-off call` you can add a link to one of our articles https://www.ombulabs.com/blog/agile/continuous-learning/kickoff-calls.html

Author: Good point! Done.

> During the kick-off call, make sure the person on the client's team assigned to the project understands what the Scrum master and the Scrum team expect from them. Some key points to cover:

Author: @etagwerker how about 'make sure the point of contact on the client's team understands' instead? I just want to make sure we emphasize it is the one person who is in charge of communicating with the team.

The Product Manager would conduct user research, identify market needs, align it with business goals and own the vision for a product. They would then communicate the requirements to the Product Owner. The Product Owner, in turn, would translate those requirements into user stories, prioritise them and work with the team to make execute on the solution.

**_The Product Owner is the point of reference for all things backlog related._** 

This is the person that brings in the client's vision, that knows what project stakeholders on the client's side want. But they'll also need to be the one to translate that into user stories and priorities within the backlog. They don't need to necessarily be the ones writing user stories, but they'll be the ones making sure stories are well defined and the team understands what needs to be done.

**_The Product Owner is responsible for ensuring the backlog is visible, transparent and clear to all._** 

> As such, the team expects the Product Owner to keep the backlog groomed and organised. They need to make sure priorities are assigned and are current. If priorities changed, this needs to be reflected in the backlog. If stories became irrelevant, they need to be removed from the backlog.

Reviewer 2: @ABizzinotto Did you see this comment?

**_The Scrum Master can be firmer during Scrum calls to make sure they fulfill their actual purpose._** 

Working with someone on the client's team might mean they are under pressure from their stakeholders and will end up gravitating towards being more of a or project manager than a Product Owner. The Scrum Master can pay attention to that and intervene when needed. 

If the Product Owner is using the Sprint Planning calls to just add that "one more story", communicate clearly and explain what the sprint planning calls are for and why is it important to the team and more efficient long term to respect the limitations set. 

If they start using retro calls as a way to pressure the team, make sure they understand retro calls are supposed to be an opportunity for learning and improvement that goes both ways. Asking why more things weren't done isn't going to help. The important thing is to evaluate what could be improved in the process and workflow so that more can be done. 

The Scrum Master can help the Product Owner understand these concepts.

**_Rethink how you talk to the Product Owner about priorities._** 

> One of the biggest problems of working with the client as the Product Owner is prioritisation. A lot of the time they will see everything as a priority. So rethinking how you talk to them about prioritisation is a good idea. Don't simply ask them to go into the backlog and prioritise stories. Instead, take some time to review the backlog with them before the next sprint planning call and ask questions such as "compared to all these other stories here, how important would it be to your business to have this done?" or "what feature here do you think would make the largest number of users happy?".

Reviewer 2: @ABizzinotto Did you see this comment?

If they are not available for the calls or not available to answer questions during a sprint, there might be delays during the project, stories might need to be reworked due to lack of understand and might even be worked in the wrong order because the backlog wasn't prioritised.

##Make it clear that there must be ONE Product Owner

Some clients will have multiple people involved in a project and, as such, several different stakeholders might attend your demo calls. The problem? In many situations, several different people might start asking for features, changes, tweaks or just "one more button".

It is important to make it clear to your client from the very beginning that the team will rely on ONE person to own the backlog, prioritise it and make sure user stories meet the requirements. And that every change or tweak that comes afterwards, if it is new functionality that wasn't specified, will need to be added as a new story.

> It is also important to enforce that during the demo calls. If the client's stakeholders start discussing new features, nice-to-haves and whatnot, the Scrum Master must intervene and make it clear that the suggestions are being noted but will be discussed further with the assigned Product Owner, who'll be responsible for making sure the user stories written for them are clear and provide the team with all the information needed and assigning a priority to them.

Reviewer 2: ```suggestion
It is also important to enforce those rules during demo calls. If the client's stakeholders start discussing new features, nice-to-haves, bells, and whistles, the Scrum Master must intervene and make it clear that suggestions are being noted. Those notes will be discussed further with the Product Owner, who will be responsible for making sure user stories are written and clear. The PO will provide the team with all the necessary details and prioritize all new stories.
```

##Conclusion

> Communicating clearly is key to making your clients better Product Owners and having in mind "red-flag" behaviours and a Scrum Master that knows how to identify them and steer the client back to the Product Owner track. These strategies should help.

Reviewer 2: ```suggestion
Communicating clearly is key to making your clients better Product Owners and having in mind "red-flag" behaviors. It is also key to count with a Scrum Master who knows how to identify red flags and steer the client back to the Product Owner track. These strategies should help.
```
